composer.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "magento/module-bundle-config",
+    "description": "Backend extension to the helpful in-browser debugging/inspection tools for the Magento 2 Front-End",
+    "require": {
+        "php": "^7.1.0|^7.2.0",
```

Comment:
2.1 is not supported anymore. But it would actually support php 7.1 according to the docs while php 7.0 is now completely abandoned and should not be used anymore

Reply:
Should not != isn't in real world ;) Just didn't notice the support was dropped for 2.1 for this module (as I saw commits to make it compatible with previous magento version earlier).

Reply:
I considered even not including PHP 7.1 as it also has reached it's [end of life](http://php.net/supported-versions.php).